tests/ops/qubit/test_matrix_ops.py

```diff
@@ -641,27 +641,24 @@ def test_decomposition_single_qubit(self):
         decomp = qml.DiagonalQubitUnitary.compute_decomposition(D, [0])
         decomp2 = qml.DiagonalQubitUnitary(D, wires=[0]).decomposition()
 
-        ph = np.exp(3j * np.pi / 4)
         for dec in (decomp, decomp2):
             assert len(dec) == 2
-            qml.assert_equal(decomp[0], qml.QubitUnitary(np.eye(2) * ph, 0))
-            qml.assert_equal(decomp[1], qml.RZ(np.pi / 2, 0))
+            qml.assert_equal(decomp[0], qml.RZ(np.pi / 2, 0))
+            qml.assert_equal(decomp[1], qml.GlobalPhase(-3 * np.pi / 4, 0))
```

Comment:
I noticed that the tests never actually test that the matrix of the original op and the matrix of the decomposed sequence are the same, can you add them?

Reply:
There is a [dedicated test](https://github.com/PennyLaneAI/pennylane/blob/fadcab6115d591b1d26758d7761d26ce1a657bd1/tests/ops/qubit/test_matrix_ops.py#L748) for that, right? :)